docs/README.md

```diff
@@ -1,149 +1,165 @@
 📢 Use this project, [contribute](https://github.com/vtex-apps/condition-layout) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Condition Layout
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-# Condition Layout
-
-The condition layout app allows to conditionally render a block given certain conditions.
+As the name implies, the Condition Layout app allows a block to be rendered if certain conditions are met.
 
 ![Screen-Recording-certo](https://user-images.githubusercontent.com/12139385/79379694-a8c99980-7f35-11ea-9f01-7021c6529332.gif)
-
-
+  
 ## Configuration
 
-
 ### Step 1 - Adding the Condition Layout app to your theme's dependencies
 
-In your theme's `manifest.json`, add the Product Price app as a dependency:
+In your theme's `manifest.json`, add the Condition Layout app as a dependency:
 
-```json
-"dependencies": {
-  "vtex.condition-layout": "1.x"
-}
+```diff
+  "dependencies": {
++   "vtex.condition-layout": "1.x"
+  }
 ```
 
-### Step 2 - Adding the Condition Layout's blocks to your theme's templates
+You are now able to use all blocks that are exported by the `condition-layout` app. Check out the full list below:
 
-The `condition-layout` app exports two kinds of block: `condition-layout.{context}` and `condition.{context}`.
+| Block name | Description |
+| -------------- | ----------------------------------------------- |
+| `condition-layout.{context}` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Top level block in which you will specify (replacing the `{context}` value in the block name) which context will be used for providing data to its child block namely `condition`. **Currently, the Condition Layout only works with the product context** therefore the top block must be `condition-layout.product`. |
+| `condition` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Defines the condition logic and the children blocks that are going to be rendered in case the predefined condition is met. |
```

Comment:
Condition blocks have a `context` "variations" as well

```suggestion
| `condition.{context}` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Defines the condition logic and the children blocks that are going to be rendered in case the predefined condition is met. |
```